### PR TITLE
API Add filter description to blog holder

### PR DESCRIPTION
**Pull request by tractorcow**
Supported by the default theme via $MetaTitle

### CODE REVIEW

**Comment by dhensby** [comment]
Shouldn't lots of this text use `_t` so it can be translated / customised?

File: code/model/Blog.php [hunk 1]
```diff
@@ -527,6 +527,11 @@ class Blog_Controller extends Page_Controller {
 		'profile/$URLSegment!' => 'profile'
 	);
 
+	private static $casting = array(
+		'MetaTitle' => 'Text',
+		'FilterDescription' => 'Text'
+	);
+
```
Comment (chillu) [inline code/model/Blog.php]: Isn't `Text` the default cast in 3.1 anyway?

Comment (tractorcow) [inline code/model/Blog.php]: Default cast isn't respected by default, only if additional actions are called on the resulting object.

E.g.

```$MyField``` will not be cast to Text.

```$MyField.Action``` will be cast to Text, and have Text::Action invoked on it.

**Comment by tractorcow** [comment]
Description has been localised, thanks @dhensby

**Pull request merged**